test/test_torch.py

```diff
@@ -3652,6 +3652,12 @@ def test_norm_fastpaths(self):
         expected = torch.pow(x.pow(3).abs().sum(1), 1.0 / 3.0)
         self.assertEqual(result, expected)
 
+        inputValuesFP32 = torch.tensor([[1, 2, 3], [-1, 1, 4]], dtype=torch.float)
```

Comment:
Why is this test added here? Why is there a totally special path for bfloat16? Please add a test to test_norm below (line 8578 or so), it has facilities to handle device-generic tests and multiple datatypes.